.github/workflows/continuous-integration.yml

```
@@ -137,14 +147,26 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
         if: ${{ steps.release-please.outputs.release_created }}
+      - name: Setup Helm
+        if: ${{ steps.release-please.outputs.release_created }}
+        uses: azure/setup-helm@v3
 
       - name: Build and publish Docker image
-        env:
-          DH_USERNAME: ${{ secrets.DOCKER_HUB_LOGIN }}
-          DH_TOKEN: ${{ secrets.DOCKER_HUB_PWD }}
-          IMAGE_NAME: "indy-vdr-proxy"
         run: |
           echo "$DH_TOKEN" | docker login -u "$DH_USERNAME" --password-stdin
           docker build -f ./packages/{{ matrix.package }}/Dockerfile -t $DH_USERNAME/$IMAGE_NAME:dev ./packages/{{ matrix.package }}
           docker push $DH_USERNAME/$IMAGE_NAME:dev
```

```suggestion
          docker push $DH_USERNAME/$IMAGE_NAME:v$RELEASE_VERSION
```

A minimal update to make DockerHub image versioning consistent with the Helm chart